readme.md

```diff
@@ -1,13 +1,13 @@
 👀*This is an early stage project under rapid development. For updates join the [Foam community Discord](https://foambubble.github.io/join-discord/g)! 💬*
 
+<img src="packages/foam-vscode/icon/FOAM_ICON_256.png" width=100 align="left">
 
 # Foam
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-64-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-64-orange.svg?style=flat-square)](#contributors-) [![Discord Chat](https://img.shields.io/discord/729975036148056075?color=748AD9&label=discord%20chat&style=flat-square)](https://foambubble.github.io/join-discord/g)
```

Comment:
why this second change? I am a bit wary of putting it inside `<!-- ALL-CONTRIBUTORS-BADGE:END -->`, feels like an auto generated code block

Reply:
If you don't keep them in the same line, one of the badges will be like:

```
figure ----| badge1
badge2
```

An alternative would be adding the alpha disclaimer to the like below the title.
Or, just ignore the comment since it still works.